pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -697,18 +698,26 @@ public long getCommitTimeoutMS(String tableName) {
 
   public static int getRealtimeTableFlushSize(TableConfig tableConfig) {
```

Comment:
May be add doc on what is being returned in what case? For example -1 if no config, and LLC if both HLC/LLC exist.

Reply:
will add comments